torch/testing/_internal/common_utils.py

```diff
@@ -1986,23 +1992,59 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         created tensor are replaced with an epsilon value if floating type, [`eps + `eps`.j] if
         complex type and 1 if integer/boolean type.
     """
+    def _modify_low_high(ranges, input_low, input_high, default_values, is_integral_dtype=False):
```

Comment:
This helper function keeps prevents redundancies in if/else conditions below (when creating tensors). Since `make_tensor` is going to be looked at again in future PRs, I believe it's useful to add helper functions to keep the logic on modifying low/high values away from other logics.